docs/modules/ROOT/pages/documentation/payara-server/server-configuration/http/protocols.adoc

```diff
@@ -13,3 +13,604 @@ commands. The protocol configurations can be found in the admin console under
 
 image::http/protocol-config.png[Configuration Page Location]
 
+== Standard Options
+
+The following standard fields can be configured for protocols:
+
+
+[[configuration-security]]
+=== Security
+
+Whether security is enabled for the protocol. When disabled, all the SSL/TLS options are ignored.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.security-enabled=[true/false]`
+
+== HTTP Options
+
+The following HTTP fields can be configured for protocols:
+
+
+[[configuration-http-server-name]]
+=== Server Name
+
+The name of the server. This will be sent as the `Server` header in responses.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.server-name-enabled=value`
+
+[[configuration-http-default-virtual-server]]
+=== Default Virtual Server
+
+The default virtual server for the protocol. If multiple virtual servers are appropriate for a request,
+this determines the priority.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.default-virtual-server=value`
+
+[[configuration-http-redirect-port]]
+=== Redirect Port
+
+The port used for SSL redirects.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.redirect-port=value`
+
+[[configuration-http-max-connections]]
+=== Max Connections
+
+The maximum number of requests per connection to keep in keep-alive mode.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.max-connections=value`
+
+[[configuration-http-timeout]]
+=== Timeout
+
+The amount of time in seconds before a connection is deemed as idle. After this, the connection will
+no longer be in keep-alive state.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.timeout-seconds=value`
+
+[[configuration-http-upload-timeout]]
+=== Upload Timeout Enabled
+
+Enables closing a connection that is reading bytes slowly after the <<configuration-http-upload-timeout-millis>> is reached.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.upload-timeout-enabled=value`
+
+[[configuration-http-upload-timeout-millis]]
+=== Connection Upload Timeout 
+
+The timeout, in milliseconds, for uploads. A value of -1 will disable it.
+
+*Asadmin Command:*
+
+`set configs.config.server-config.network-config.protocols.protocol.${protocol-name}.http.connection-upload-timeout-millis=value`
+
+[[configuration-http-request-timeout]]
+=== Request Timeout
+
+The timeout, in seconds, for requests. A value of -1 will disable it.
```

Comment:
The second sentence should be in an admonition, like:

NOTE: To disable this timeout, set the value to `-1`